simpl/chronos.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2011-2015 Rackspace US, Inc.
+# All Rights Reserved.
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Simpl time utilities.
+
+Use for consistent date/time conversion and date/time formatting in APIs.
+
+In code, get current time formatted as a string using
+`chronos.get_time_string()`. Date/times are always UTC. We leave it to clients
+to convert to local time.
+
+Our APIs consitently return date/time in the API_FORMAT format.
+"""
+
+import datetime
+import time
+
+API_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
If this format, and these utilities, comply with [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601), maybe leave a note/comment/documentation about that.